chat/Cargo.toml

```diff
@@ -15,6 +15,7 @@ gpui.workspace = true
 iroh = "0.26.0"
 pkarr = "2.2.0"
 postcard = "1.0.10"
+pod2 = { path = "../pod2" }
```

Comment:
Can you import it from the workspace? See how I did it for the "assets" crate

Reply:
yes